### PR TITLE
Add support for N-D intersects '&&&' operator

### PR DESCRIPTION
**Pull request by dlbrittain**
Add support for N-D spatial intersection for querying geometries in bounding boxes
See here for info:
http://postgis.net/docs/geometry_overlaps_nd.html
List of all operators in Postgis
http://postgis.net/docs/reference.html#Operators

### CODE REVIEW

**Comment by adrien-berchet** [comment]
Hello @dlbrittain 
This looks nice, thank you very much for this!

File: geoalchemy2/comparator.py [hunk 1]
```diff
@@ -220,3 +221,10 @@ def distance_box(self, other):
         geometries.
         """
         return self.operate(DISTANCE_BOX, other, result_type=DOUBLE_PRECISION)
+
+    def intersects_3d(self, other):
```
Comment (adrien-berchet) [inline geoalchemy2/comparator.py:225]: I am just wondering about the name, shouldn't it be `intersects_nd`?

Comment (adrien-berchet) [reply geoalchemy2/comparator.py]: :+1:

**Comment by adrien-berchet** [comment]
Thank you very much!
Do you need a new release?

**Comment by dlbrittain** [comment]
It would be very helpful indeed! Thank you!

**Comment by adrien-berchet** [comment]
The version 0.9.2 was released :)

**Pull request merged**